### ISSUE TITLE
System tray menu is not translated into Finnish. Which is very bad - like some apocalypse level stuff.

### ISSUE THREAD

**Issue by olavinto** [AdguardTeam/AdGuardVPNForWindows#291]
### Issue Details

- **AdGuard VPN version:**
  - 1.3.581.0 (nigthly)
- **Operating system and version:**
  - Windows 11 Insider Preview

### Expected Behavior

A user selects Finnish as the language of the application. Some magical shit happens and they experience a shocking revelation: AdGuard VPN suddenly appears on their desktop gloriously translated into Finnish. They are obviously extremely happy because of this and write a super positive review mentioning that "The app is the best in the whole multiverse and it's even translated into Finnish".

### Actual Behavior

The user selects Finnish as the language of the application. Some magical shit happens and they experience a revelation: AdGuard VPN is only partly translated into Finnish. They are super confused (obviously) and wonder what the hell is this crap in the system tray menu. They are (again, obviously) extremely disappointed, uninstall the application and write a super negative review mentioning that "The app was something unpronounceable and only parts of it were translated into Finnish. I am never-ever coming back to it and other bad things. My life is in ruins.".

### Screenshots

Here is a beautifully crafted (took me like hours to set this take up) screenshot of the untranslated system tray menu. As you can clearly see, it remains in English. And yes, to you, you customer service agent with that list of obvious questions to go through: I have activated Finnish in the settings! Or have I? What if I didn't? Maybe someone else did it? Think about that. Or don't. It's up to you. Or maybe it isn't. Who knows. I'm now so confused that I don't even know what I came here to do in the first place.

![avpntrayuntranslated](https://user-images.githubusercontent.com/23163218/177846399-f83c3aaf-3e94-4213-a6ed-86d69f8c4eb4.png)

Disclaimer: Unless otherwise not apparent to some of our more serious viewers, this post is not supposed to be taken very seriously.

**Comment by mozartinio** [comment]
@olavinto Am i right that you change your language in Windows settings ? (not inside VPN app)

**Comment by olavinto** [comment]
> @olavinto Hello! Firstly want to thank you for such an amazing issue details;) Am i right that you change your language in Windows settings ? (not inside VPN app)

Oh man, that was the other thing I should have mentioned to not ask me about! :)

Check this thing out:

![avpntrayuntranslated2](https://user-images.githubusercontent.com/23163218/178010431-b7061c30-7054-4116-89b7-660cba430184.png)

**Comment by olavinto** [comment]
Actually, to be more specific, I have custom language/locale settings in Windows (basically English with Finnish units and stuff). I also have the Finnish language pack installed but not active.

![avpntrayuntranslated3](https://user-images.githubusercontent.com/23163218/178011767-cfe24dfe-2a22-4eca-b48e-a11896564c50.png)

**Comment by olavinto** [comment]
@mozartinio The menu is still in English in the Nightly v2.0.709.0. Should it be fixed already?

**Comment by mozartinio** [comment]
@olavinto we know about and already have nightly 2.0.718 version with fix

**Comment by olavinto** [comment]
> @olavinto we know about and already have nightly 2.0.718 version with fix

I just noticed that the menu is acting a bit strangely. I have two Windows systems. One is in English (Window 11 Dev channel) and the other in Finnish (Windows 10 Release). In the Windows 11 machine I have AdGuard VPN set to Finnish and the apps UI is translated but the tray menu remains in English. In the Windows 10 machine the tray menu is also translated into Finnish. Application version on both is 2.1.912.0 (I've also done many version updates on both systems after this issue was closed).
